### ISSUE TITLE
New function that may fix targeting issues in Dynamis

### ISSUE THREAD

**Issue by Kuresu** [EasyFarm/EasyFarm#238]
Hey buddy, again first of all props for your little program.

It has really spared me alot of adult time on some of the grind haha.

I'm currently working on dynamis, and i'm experiencing some very trying issues with targeting that i can't seem to find a fix for.
Due to the sheer amount and variety of mobs, targeting will sometimes screw up, where it targets a mob, but in reality, the game has another mob in sight, making the character go into an engaged limbo, where nothing is attacked.

I've tried every combination of fixes in terms of built in aggro order etc in the app, but nothing seems to work.

So, what i usually do is check every now and then, and when the character is stuck, i disengage the battle, and he will automatically reengage the right mob and start attacking again.

So i was wondering if you could add a function, in which i can determine fx.

Every 3mins, main character will disengage from battle and then automatically reengage.
This would allow survivability in Dynamis for a longer period of time.

Since i can imagine, it is hard to make adjustments to targetting due to the infinite number of variables that exist in terms of mob placement, distance, amount etc.

**Comment by Mykezero** [comment]
There's definitely a lot of problems with targeting at the moment. 

That's a very good suggestion: It might be possible to somehow track how long we haven't engaged / fought the mob for, and use that to perform a reset. 

Maybe modifying the code which sets the target mob, to also track how long that mob has been our target for, and modifying one of the other states to perform the disengage. 

Definitely going to re-open this issue since this is a problem that needs to be solved. 

Sorry about not leaving a comment sooner! ^^;

**Comment by Kuresu** [comment]
Hey Myke.

Np i partially solved the issue by figuring out, that the autotarget button in the config menu isn't as effective as /autotarget off.

Using the actual command actually solved alot of the targeting issues, but sometimes i'll still find my character dead due to targeting issues, albeit they're much rarer now.

But as said, i think this kind of fix would solve the remainder of the targeting problems experienced in situations where there is alot of mobs to choose from.

Still an awesome program none the less.

**Comment by Mykezero** [comment]
Not too sure why I didn't think of this sooner: On start, the program could send the /autotarget off command. That might save a lot of headaches!

**Comment by Kuresu** [comment]
Yes! that would actually solve alot of our rutinely problems haha.
